client/src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ReactiveFormsModule } from '@angular/forms';
+import { RouterModule, Routes } from '@angular/router';
+
+import { SharedModule } from '../shared/shared.module';
+
+import { LoginFormComponent } from './login-form/login-form.component';
+
+const routes: Routes = [
+  { path: 'login', component: LoginFormComponent }
```

Comment:
i think this route should be moved to app routing module

Reply:
I moved this logic to app routing module and I removed it from aut.module.ts